packages/suite-desktop/src/modules/tor.ts

```diff
@@ -48,6 +53,46 @@ const load = async ({ mainWindow, store }: Dependencies) => {
     const getProxySettings = (shouldEnableTor: boolean) =>
         shouldEnableTor ? { proxy: `socks://${address}` } : { proxy: '' };
 
+    const handleTorProcessStatus = (status: TorProcessStatus) => {
+        let type: TorStatusEventType;
+
+        if (!status.process) {
+            type = TOR_STATUS_EVENT_TYPE.Disabled;
+        } else if (status.isBootstrapping) {
```

Comment:
switch

Reply:
I do not think a switch is the best solution in this case. 
Taking into account that all the cases are boolean, switch are good when you want to match a string for example.
It would have to be using the `switch(true)` which I don't really like. Or did you have something different in mind?

Reply:
True, I believe that I saw something else here in the morning 😆